modules/checklist/assets/js/components/checklist.js

```diff
@@ -0,0 +1,26 @@
+import { ThemeProvider, DirectionProvider } from '@elementor/ui';
+import Launchpad from './launchpad';
+import { useEffect, useState } from 'react';
+
+const Checklist = ( props ) => {
+	const [ isOpen, setIsOpen ] = useState( true );
+
+	useEffect( () => {
+		elementor.on( 'elementor/editor/panel/checklist/clicked', () => setIsOpen( ! isOpen ) );
+	}, [] );
+
+	return isOpen && (
+		<DirectionProvider rtl={ props.isRTL }>
+			<ThemeProvider colorScheme="light">
```

Comment:
I think I would expect to see these in the app file, these usually go at the topmost level and wrap everything

Reply:
@louiswol94 @ronenelementor indeed usually it is like that, in this case, to put App file outside, I'll have also to move editor-v-2.js file outside. WDYT?